src/app/components/tieredmenu/tieredmenu.ts

```diff
@@ -222,7 +222,7 @@ export class TieredMenuSub {
 
     @ViewChild('sublist', { static: true }) sublistViewChild: ElementRef;
 
-    constructor(@Inject(DOCUMENT) private document: Document, public el: ElementRef, public renderer: Renderer2, private cd: ChangeDetectorRef,@Inject(forwardRef(() => TieredMenu)) public tieredMenu: TieredMenu) {}
+    constructor(public el: ElementRef, public renderer: Renderer2, private cd: ChangeDetectorRef, @Inject(forwardRef(() => TieredMenu)) public tieredMenu: TieredMenu) {}
```

Comment:
@denny99 why we need to  remove `@Inject(DOCUMENT) private document: Document` this line

Reply:
It was just unused, so i cleaned up the code

Reply:
@denny99  ok